### PR TITLE
update Set-SpoSite

### PR DESCRIPTION
**Pull request by sijeee**
add `-ClearGroupId` parameter

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 1]
```diff
@@ -1460,6 +1460,21 @@ Accept wildcard characters: False
 
 ```
 
+
+### -ClearGroupId
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:1464]: >ClearGroupId [](http://example.com/codeflow?start=5&length=12)

you need to add this to the SYNTAX section as well. Scroll to top of page.

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 2]
```diff
@@ -1460,6 +1460,21 @@ Accept wildcard characters: False
...
+This parameter allows you to remove the assigned M365 group id on a site, when the group is permanently deleted.
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:1465]: >M365 group id [](http://example.com/codeflow?start=49&length=13)

Microsoft 365 group ID

**Review by samkabue — CHANGES_REQUESTED** [review]
:clock1:

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md (hunk 1 above)
Comment (sijeee) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:1471]: added

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md [hunk 3]
```diff
@@ -101,6 +101,13 @@ Set-SPOSite [-Identity] <SpoSitePipeBind>
  [<CommonParameters>]
 ```
 
+### ClearGroupId
+```powershell
+Set-SPOSite [-Identity] <SpoSitePipeBind>
+ [-ClearGroupId]
+ [<CommonParameters>]
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOSite.md:108]: keep as 1 line as this is short.

```
Set-SPOSite [-Identity] <SpoSitePipeBind> [-ClearGroupId] [<CommonParameters>]
```

**Review by samkabue — APPROVED** [review]
:shipit:

**Pull request merged**